bitnami/rabbitmq/values.yaml

```diff
@@ -224,6 +224,9 @@ service:
   ##
   managerPort: 15672
 
+  ## Service extra labels
+  extraLabels: {}
```

Comment:
Even though I agree using `extraLabels` is technically correct, we're using `service.labels` to add extra labels on other charts (see Redis or NGINX Ingress Controller for instance). Could you we please follow the same approach for consistency with these other charts?